Modules/_ssl.c

```diff
@@ -2851,11 +2861,13 @@ _ssl__SSLSocket_verify_client_post_handshake_impl(PySSLSocket *self)
 @critical_section
 @getter
 _ssl._SSLSocket.session
+
+The underlying SSLSession object.
```

Comment:
FTR, this was previously _"Get / set SSLSession."_.

Reply:
Yeah, but that's not particularly useful. I thought this made more sense.